src/scenes/home/gala/gala.css

```diff
@@ -1,38 +1,15 @@
-.header {
+/*Hero*/
+.hero {
+  background-image: url('./portland19.jpg');
+  height: 100vh;
```

Comment:
Doesn't seem like the `height: 100vh` rule needs to be here. Toggling it on/off doesn't change anything.  If that's the case, this rule should be removed.